vulnerabilities/import_runner.py

```diff
@@ -32,65 +32,34 @@
 from vulnerabilities import models
 from vulnerabilities.models import Advisory
 from vulnerabilities.data_source import AdvisoryData
+from vulnerabilities.data_source import DataSource
 
 logger = logging.getLogger(__name__)
 
 
 class ImportRunner:
     """
     The ImportRunner is responsible for inserting and updating data about vulnerabilities and
-    affected/unaffected/fixed packages in the database. The two main goals for the implementation
-    are correctness and efficiency.
+    affected/unaffected/fixed packages in the database. The main goal for the implementation
+    is correctness
 
     Correctness:
         - There must be no duplicates in the database (should be enforced by the schema).
         - No valid data from the data source must be skipped or truncated.
-
-    Efficiency:
-        - Bulk inserts should be used whenever possible.
-        - Checking whether a record already exists should be kept to a minimum
-        (the data source should know this instead).
-        - All update and select operations must use indexed columns.
     """
 
-    def __init__(self, importer: models.Importer):
+    def __init__(self, importer: DataSource):
```

Comment:
This should be an Importer, not a data source IMHO

Reply:
Correct. I intend to rename all DataSource to Importer but that's for next PR, out of context for this PR.

Reply:
https://github.com/nexB/vulnerablecode/issues/601